scripts/get_native_libraries.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+import urllib.request
+import shutil
+import ssl
+import zipfile
+
+from os import path, remove
+
+
+def download_native_libraries(name: str, version: str, destination: str):
+    ssl._create_default_https_context = ssl._create_unverified_context
+    jni_libs = 'android/src/main/jniLibs'
+
+    to_be_copied = {}
+    to_be_copied[
+        f'tmp/x86_64-unknown-linux-gnu/x86_64-unknown-linux-gnu/{name}.h'
+    ] = f'{destination}/{name}.h'
+    to_be_copied[
+        f'tmp/x86_64-apple-darwin/x86_64-apple-darwin/release/libcosmian_{name}.dylib'
+    ] = f'{destination}/libcosmian_{name}.dylib'
+    to_be_copied[
+        f'tmp/x86_64-unknown-linux-gnu/x86_64-unknown-linux-gnu/release/libcosmian_{name}.so'
+    ] = f'{destination}/libcosmian_{name}.so'
+    to_be_copied[
+        f'tmp/x86_64-pc-windows-gnu/x86_64-pc-windows-gnu/release/cosmian_{name}.dll'
+    ] = f'{destination}/cosmian_{name}.dll'
+    to_be_copied[
+        f'tmp/android/armeabi-v7a/libcosmian_{name}.so'
+    ] = f'{jni_libs}/armeabi-v7a/libcosmian_{name}.so'
+    to_be_copied[
+        f'tmp/android/arm64-v8a/libcosmian_{name}.so'
+    ] = f'{jni_libs}/arm64-v8a/libcosmian_{name}.so'
+    to_be_copied[
+        f'tmp/android/x86/libcosmian_{name}.so'
+    ] = f'{jni_libs}/x86/libcosmian_{name}.so'
+    to_be_copied[
+        f'tmp/android/x86_64/libcosmian_{name}.so'
+    ] = f'{jni_libs}/x86_64/libcosmian_{name}.so'
+    to_be_copied[
+        f'tmp/x86_64-apple-darwin/universal/release/libcosmian_{name}.a'
+    ] = f'ios/libcosmian_{name}.a'
```

Comment:
Why mutate an empty dict, the following is not working?
```
thisdict =	{
  "brand": "Ford",
  "model": "Mustang",
  "year": 1964
}
```

Reply:
true!